src/pages/en/core-concepts/astro-components.md

```diff
@@ -275,10 +275,15 @@ In Astro, you use the standard `kebab-case` format for all HTML attributes inste
 
 In Astro, you can use standard HTML comments where JSX would use JavaScript style comments.
 
-```html title="example.astro"
+```astro title="example.astro"
 <!-- HTML comment syntax is valid in .astro files -->
```

Comment:
```suggestion
---
---
<!-- HTML comment syntax is valid in .astro files -->
```
It might be good to show that these comments are in the templating section of an Astro file.